torch_em/segmentation.py

```diff
@@ -1,7 +1,7 @@
 import os
 from glob import glob
 
-import torch
+import torch.utils.data
```

Comment:
nothing stopping us from importing both
```suggestion
import torch
import torch.utils.data
```